test/functional/Java16andUp/src/org/openj9/test/jep389/downcall/InvalidDownCallTests.java

```diff
@@ -0,0 +1,387 @@
+package org.openj9.test.jep389.downcall;
+
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.testng.AssertJUnit;
+import static org.testng.Assert.fail;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodType;
+import jdk.incubator.foreign.CLinker;
+import static jdk.incubator.foreign.CLinker.*;
+import jdk.incubator.foreign.FunctionDescriptor;
+import jdk.incubator.foreign.ValueLayout;
+import jdk.incubator.foreign.MemoryLayout;
+import jdk.incubator.foreign.MemoryLayouts;
+import jdk.incubator.foreign.MemoryAddress;
+import jdk.incubator.foreign.LibraryLookup;
+import static jdk.incubator.foreign.LibraryLookup.Symbol;
+
+/**
+ * Test cases for JEP 389: Foreign Linker API (Incubator) DownCall for primitive types,
+ * which verifies the majority of illegal cases including the mismatch between the type
+ * and the corresponding layout, the inconsisitency of the arity, etc.
+ */
+@Test(groups = { "level.sanity" })
+public class InvalidDownCallTests {
+	private static boolean isWinOS = System.getProperty("os.name").toLowerCase().contains("win") ? true : false;
```

Comment:
Please remove `? true : false` (it's simply redundant).

Reply:
Agreed and fixed as suggested above.